docs/09.EXT-REFERENCE-ARG.md

```diff
@@ -138,33 +162,35 @@ static jerry_value_t my_external_handler (const jerry_value_t function_obj,
   char required_str[16];
   double optional_num = 1234.567;  // default value
 
-  // "mapping" defines the steps to transform input arguments to C variables:
+  /* "mapping" defines the steps to transform input arguments to C variables. */
   const jerryx_arg_t mapping[] =
   {
-    // `this` is the first value. No checking needed on `this` for this function.
+    /* `this` is the first value. No checking needed on `this` for this function. */
     jerryx_arg_ignore (),
 
     jerryx_arg_boolean (&required_bool, JERRYX_ARG_NO_COERCE, JERRYX_ARG_REQUIRED),
     jerryx_arg_string (&required_str, sizeof (required_str), JERRYX_ARG_NO_COERCE, JERRYX_ARG_REQUIRED),
     jerryx_arg_number (&optional_num, JERRYX_ARG_NO_COERCE, JERRYX_ARG_OPTIONAL),
   };
 
-  // Validate and transform:
+  /* Validate and transform. */
   const jerry_value_t rv = jerryx_arg_transform_this_and_args (this_val,
                                                                args_p,
                                                                args_count,
                                                                mapping,
-                                                               ARRAY_LENGTH (mapping));
+                                                               4);
 
   if (jerry_value_has_error_flag (rv))
   {
-    // Handle error
+    /* Handle error. */
     return rv;
   }
 
-  // Validated and tranformed successfully!
-  // required_bool, required_str and optional_num can now be used.
-  // ...
+  /*
+   * Validated and transformed successfully!
+   * required_bool, required_num and optional_num can now be used.
```

Comment:
There is a misunderstanding. The way the comment was formatted (with `/* ... */`) was OK with me. The problem is that the comment text itself has changed. Somehow required_str became required_num.